src/main/java/beans/RepeatRequestBean.java

```diff
@@ -9,22 +9,31 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class RepeatRequestBean extends SessionBean {
+    private String ix;
```

Comment:
ah i see. how about calling it `index` or something with a comment as to why we have to do this